src/PackageManagement.VisualStudio/Utility/EnvDTEProjectUtility.cs

```diff
@@ -625,7 +640,9 @@ public static string GetTargetFrameworkString(EnvDTEProject envDTEProject)
                 ||
                 string.Equals(envDTEProject.Kind, NuGetVSConstants.FsharpProjectTypeGuid, StringComparison.OrdinalIgnoreCase)
                 ||
-                string.Equals(envDTEProject.Kind, NuGetVSConstants.JsProjectTypeGuid, StringComparison.OrdinalIgnoreCase))
+                string.Equals(envDTEProject.Kind, NuGetVSConstants.JsProjectTypeGuid, StringComparison.OrdinalIgnoreCase)
+                ||
+                string.Equals(envDTEProject.Kind, NuGetVSConstants.CpsProjectTypeGuid, StringComparison.OrdinalIgnoreCase))
```

Comment:
The new code becomes:

    int pFound; 
    uint itemId; 
    VSDOCUMENTPRIORITY[] priority = new VSDOCUMENTPRIORITY[1];
    int hr = vsProject.IsDocumentInProject(path, out pFound, priority, out itemId); 
    return ErrorHandler.Succeeded(hr) && pFound == 1 && priority[0] >= VSDOCUMENTPRIORITY.DP_STANDARD;